_docs/useful-links.md

```diff
@@ -39,13 +39,13 @@ The following is a list of links for data providers and web applications develop
 ## Submitting Data
  - [**Deployment Registration**](https://gliders.ioos.us/providers)
 
-This link will redirect you to the GliderDAC providers page. From there you can login and create deployments. See the wiki section on [New Deplyment Registration](/ioosngdac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.
+This link will redirect you to the GliderDAC providers page. From there you can login and create deployments. See the wiki section on [New Deplyment Registration](/glider-dac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.
```

Comment:
We can't point to a separate wiki located elsewhere.  Also there's a typo in 'deployment.'
Change the sentence "See the wiki..." to: "See the section on [New Deployment Registration](https://ioos.github.io/glider-dac/ngdac-netcdf-file-submission-process.html#new-deployment-registration) for more details.